.travis.yml

```diff
@@ -1,15 +1,15 @@
-language: java
-jdk: oraclejdk8
+language: node_js
+node_js: node
 sudo: false
 addons:
   apt:
     packages:
     - jq
 env:
   global:
-  - SPEC_FILE: https://www.thebluealliance.com/swagger/api_v3.json
+  - SPEC_FILE: https://raw.githubusercontent.com/tweirtx/the-blue-alliance/master/static/swagger/api_v3.json
```

Comment:
Did you intend to commit this to master...? Or was this just done until TBA merges your spec changes?

Reply:
That was temporary, if you look at the file it's back to the regular one

Reply:
Hadn't gotten that far yet.  👍